Makefile

```diff
@@ -1,4 +1,10 @@
+default: edit
+
 ten-simple-rules-dockerfiles.pdf: ten-simple-rules-dockerfiles.Rmd
 	Rscript -e 'rmarkdown::render("$<")'
 
+edit:
+	PASSWORD=simple
```

Comment:
Maybe this should not be in plain sight, even if it's just for development?

Reply:
I think we don't ever suggest to run the container other than somewhere locally, and I would not even expect someone other than the authors to do so, so I'd say we leave that it. More important things to work on ;-)